website/api/generating-proofs/local-proving.md

```diff
@@ -6,14 +6,75 @@ The [feature flags] include options to toggle between CPU and GPU proving.
 
 ## Local vs. Remote Proving
 
-For most use cases, we recommend remote proving using [Bonsai].
-The primary reason to consider using local proof generation for your application is to manage private data.
+For most use cases, we recommend remote proving using [Bonsai], our scalable and highly parralel proving service.
 
-> Whoever is generating the proofs can see all private information involved.
+Proving with **private data** is a key reason to consider using local proof generation, as whoever is generating the proofs can see all private information involved.
+With local proof generation your private data never leaves your machine.
 
-_With local proof generation, you can generate proofs about your private data, while your private data never leaves your machine._
+## Proving hardware
+
+RISC Zero's proving system for multiple hardware targets.
+
+:::info
+The Groth16 prover currently _only_ works on x86 architecture, and so Apple Silicon is _currently unsupported_ (even via Docker).
+
+You can find out more info in the relevant issues [here](https://github.com/risc0/risc0/issues/1520) and [here](https://github.com/risc0/risc0/issues/1749).
```

Comment:
I'd prefer to stick with our link conventions -- i.e., putting links at the bottom of the page and avoid use of `[here]` 

```suggestion
You can find out more info in the relevant issues [#1520] and [#1749].
```